extensions/pom.xml

```diff
@@ -18,8 +18,10 @@
     <description>Mule extensions that augment the core Mule functionality through the Mule Extension API</description>
 
     <properties>
+        <generateDocs>false</generateDocs>
         <mule.extensions.maven.plugin.version>1.0.0-SNAPSHOT</mule.extensions.maven.plugin.version>
         <mule.metadata.version>1.0.0-SNAPSHOT</mule.metadata.version>
+        <extension.documentation.maven.plugin.version>1.0.0-SNAPSHOT</extension.documentation.maven.plugin.version>
```

Comment:
a) this should be in a constant
b) didn't you talked to me today about this being part of the extensions packager?